shopelectro/management/commands/update_in_pack.py

```diff
@@ -0,0 +1,29 @@
+import logging
```

Comment:
maybe separated command is not necessary?
I think it has no sense to launch pack sync with CLI